zhmccli/_cmd_partition.py

```diff
@@ -439,6 +441,42 @@ def partition_console(cmd_ctx, cpc, partition, **options):
                                                       options))
 
 
+@partition_group.command('mountiso', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('PARTITION', type=str, metavar='PARTITION')
+@click.option('--imagefile', type=str, required=True,
+              help='The path and name of the ISO image.')
+@click.option('--imageinsfile', type=str, required=True,
+              help='The path and name of the ins file.')
+@click.pass_obj
+def partition_mount_iso(cmd_ctx, cpc, partition, **options):
+    """
+    Mount an ISO image to a partition.
+
```

Comment:
Add: "The boot options of the partition will be set to boot from the ISO file."

Reply:
This can be set by specifying the new '--boot'/'-b' option.
'

Reply:
Ok.